lab-jacob/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "build-tools",
+  "version": "0.1.0",
+  "description": "second lab assignment learning about build tools",
+  "main": "greet.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/JacobKnaack/lab-02-build-tools.git"
+  },
+  "author": "Jacob Knaack",
+  "license": "ISC",
```

Comment:
Make sure this is the correct license, ISC is fine, just something to keep in mind.